meson.build

```
@@ -0,0 +1,23 @@
+project('ucsd-psystem-vm', 'c')
```

```suggestion
project('ucsd-psystem-vm', 'c', meson_version: '>=0.58.0')
```

You use env.prepend multiple times for the same env variable (newest feature AFAICT), so setting a minimum version here will ensure that e.g. people using Debian Bullseye (stable) with a slightly too old version of meson from `apt install meson`, get an error message telling them they need to upgrade e.g. by getting it from pip instead.

(It will also print compatibility warnings if you use features that aren't available in the minimum supported version of meson that you declared.)

---

Also a good idea!  Will definitely fix on the next pass.